src/ui/ui_exports/ui_export_defaults.js

```diff
@@ -46,9 +46,10 @@ export const UI_EXPORT_DEFAULTS = {
       'ui/vis/request_handlers/none'
     ],
     visResponseHandlers: [
-      'ui/vis/response_handlers/basic',
+      'ui/vis/response_handlers/vislib',
```

Comment:
What is the difference between vislib and legacy? Maybe add some comments in the respective `response_handlers/*` that specify what the response gets converted into so devs can figure out why they exist and when they should be used.

Reply:
generally, none of them should be used anymore ....
`vislib` is only used by ... well `vislib` :) we called it basic before ... we'll write some migration docs that will mention the rename and that this is now deprecated and will be removed with 7.0
`legacy` is the old `default` .... its used by `table` visualization internally still, and probably a lot of 3rd party visualizations are using this. Again we will mention the rename in migration docs as well that this one is deprecated as well. We will also try to write something migration from the old table format to the new one.

Reply:
sounds good. Please include what the handlers actually do. Legacy and vislib are vague - The documentation should answer the question... How does the handler process the response?